src/protojson/CMakeLists.txt

```diff
@@ -1,8 +1,11 @@
 add_library(protojson SHARED)
 
 target_sources(protojson PRIVATE
+    JobErrorMsg.cc
+    JobReadyMsg.cc
 	ScanTableInfo.cc
     UberJobMsg.cc
+    WorkerCzarComIssue.cc
```

Comment:
Check the indentation of the lines in this section. Line number 6 (for `ScanTableInfo.cc`) has 4 more spaces .